solr/core/src/test/org/apache/solr/util/TestUtils.java

```diff
@@ -294,4 +294,23 @@ public void testMergeJson() {
     assertEquals(
         2L, Utils.getObjectByPath(sink, true, List.of(DEFAULTS, COLLECTION_PROP, NRT_REPLICAS)));
   }
+
+  @SuppressWarnings({"unchecked"})
+  public void testToJson() {
```

Comment:
I would prefer 3 tests here:
edge case 0
edge case -1
standard case

I've never been a fan of mixing test cases within a single one (even if it's simple)

Reply:
I did the required thanks. Thanks for the suggestion!